HLTrigger/HLTcore/interface/TriggerExpressionData.h

```diff
@@ -87,15 +87,15 @@ namespace triggerExpression {
          edm::InputTag const& l1tResultsTag,
          bool l1tIgnoreMaskAndPrescale,
          bool doThrow,
-         edm::ConsumesCollector&& iC)
+         edm::ConsumesCollector&& cc)
```

Comment:
FYI we changed this so you can pass a copy of a ConsumesCollector which avoids some weirdnesses with using `&&`.